docs/file-conventions/-client.md

```diff
@@ -22,6 +22,6 @@ console.log(supportsVibrationAPI);
 // client: true | false
 ```
 
-See [Route Module][routemodule] for more information.
+See [Route Module][route-module] for more information.
```

Comment:
just fyi, I do the gross `routemodule` instead of `route-module` so that my editor autocompletes links better 🫣

Reply:
I was just trying to keep them in line with how we do it mostly in the rest of the docs (it's not fully like this though)